mathics/builtin/layout.py

```diff
@@ -16,7 +16,13 @@
 from mathics.builtin.makeboxes import MakeBoxes
 from mathics.builtin.options import options_to_rules
 from mathics.core.atoms import Real, String
-from mathics.core.builtin import BinaryOperator, Builtin, Operator
+from mathics.core.builtin import (
+    BinaryOperator,
```

Comment:
WIth the changes to use PostfixOperator and PrefixOperator (thank you very much), the "BinaryOperator" is no longer imported.